README.md

```diff
@@ -1370,19 +1370,15 @@ Additional applications and services which offer backups include:
 
 # Wi-Fi
```

Comment:
nb: we should add information on possible risks of wireless hardware firmware and drivers; reducing the overall radio profile will generally lead to better security and privacy outcomes.